api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -1225,6 +1225,25 @@ public interface Component extends ComponentBuilderApplicable, ComponentLike, Ex
     return text(String.valueOf(value), color, decorations);
   }
 
+  /*
+   * --------------------------
+   * ---- VirtualComponent ----
+   * --------------------------
+   */
+
+  /**
+   * Creates a virtual component with a value.
+   *
+   * @param virtual the value
+   * @return a virtual component
+   * @since 4.13.0
+   */
+  @Contract(value = "_ -> new", pure = true)
+  static @NotNull VirtualComponent virtual(final @NotNull VirtualComponentHolder<?> virtual) {
```

Comment:
hm, maybe worth adding a StyleBuilderApplicable... varargs overload, and a `Style` overload, as kezz mentioned for the translatable fallback PR?